docs/pages/project/contributing/contributing-cli.md

```diff
@@ -128,6 +128,27 @@ When designing for the command line interface, ask and consider the following qu
 
 You will need to provide a short and long description of the command for the help pages and also for the Meshery Documentation.
 
+## Refactoring Mesheryctl Code
```

Comment:
Are we asking contributors to refactor code? What does Refactoring mesheryctl code have to do with how contributors should use centralized structs?